ansible/roles/sap_hana_install/tests/install/run-sap_hana_install-install-tests.py

```diff
@@ -4,34 +4,35 @@
 import sys
```

Comment:
This is not one of the files of the glue script, does not mean that fixes has not to be accepted, but I'm not sure what exactly this script is about. So I'm not sure that we have to apply the same linting rules here too

Reply:
> This is not one of the files of the glue script, does not mean that fixes has not to be accepted, but I'm not sure what exactly this script is about. So I'm not sure that we have to apply the same linting rules here too

This is an unused import.